### DISCUSSION TITLE
Data of "plug in status"

### DISCUSSION THREAD

**Discussion by swotai** [teslamate-org/teslamate#1463] [Ideas]
On the front page of the app (not grafana) there's a small icon that says whether the car is plugged in. I'm wondering if that info is also available on the postgres db / grafana. 

I'm looking into setting a reminder in case I forget to plug it in on Thurs-Sat night. I think it should be quite easy with grafana + slack.

**Comment by LelandSindt** [comment]
You could write something that connects to [MQTT](https://docs.teslamate.org/docs/integrations/mqtt) subscribes to `teslamate/cars/$car_id/geofence` and `teslamate/cars/$car_id/plugged_in`.  If `plugged_in == false` and `geofence == "home"` and `time > 7pm` then ... `send alert`
